kview/src/main/java/dev/ikm/komet/kview/mvvm/view/journal/JournalController.java

```diff
@@ -1001,8 +1015,17 @@ private void createConceptWindow(ConceptFacade conceptFacade,
             preferences.put(ENTITY_NID_TYPE, nidTextEnum.name());
         }
 
+        // TODO: Test and Fixme this takes a snapshot of the navigator's view coordinate
+        //       and makes a new view properties.
+        ObservableViewNoOverride observableViewNoOverride = new ObservableViewNoOverride(
+                getNavigatorNode()
+                        .getViewProperties()
+                        .nodeView()
+                        .getViewCoordinate());
```

Comment:
Could you do me a favor? Please Remove lines **1020-1024** (leave the comments above).
The comments explain a TODO, the code actually broke the create concept window. I just encountered it during some additional testing.